linalg/sparsemat.cpp

```diff
@@ -1055,7 +1063,7 @@ void SparseMatrix::AbsMultTranspose(const Vector &x, Vector &y) const
    else
    {
       MFEM_VERIFY(Device::IsDisabled(), "transpose action on device is not "
```

Comment:
Change the condition here to be the inverse of the one in `EnsureMultTranspose`.

Reply:
`Device::Allows(Backend::CPU_MASK)`?

Reply:
This is not the inverse. `Backend::CPU` is always enabled, so `Device::Allows(Backend::CPU_MASK)` is always true. Just use `!Device::Allows(~Backend::CPU_MASK)` or `Device::Allows(~Backend::CPU_MASK) == false`.

Reply:
That makes sense, thanks. I'm not familiar with `Device::Allows`.

Reply:
It's very simple: `Device::Allows(mask)` returns `(configured_backends_mask | mask)` where `configured_backends_mask` is a bitwise or of all currently configured (enabled) backends.

Reply:
I updated it — hopefully it's ok now.